Update translation instructions (#858)
jamulussoftware/jamulussoftware.github.io: _translator-files/README.md

# Website docs translation:

The recommended way to translate the website documentation is by using [Weblate](https://hosted.weblate.org/projects/jamulus/). If you don't already have one, we recommend that you create a GitHub account and log in to Weblate using that. If you are comfortable using GitHub and .po file editors, and still want to contribute your translation via a pull request, the files for each language are contained in: `_translator-files/po/LANGUAGE/`. You can find a guide on .po file editors [here](https://jamulus.io/kb/2022/08/04/Translating-po-files.html). Please make your PR against the `next-release` branch.

All the Jamulus project files (application and website) are listed together on Weblate. The files ("components") corresponding to the application itself are "Jamulus app" and "Windows Installer". All the rest correspond to the website.

**Some tips regarding translations:**

> - Change the `lang: en` parameter in the files containing it at the beginning of the file to your language code (it, fr, de, etc.). Remember that if you are translating a regional variation (e.g. `pt_BR`), that's what you must use. Also make sure the `permalink: [parameter]` (e.g. /wiki/Client-Troubleshooting) stays the same

Reviewer: Full stop missing at the end of this sentence.  May as well add it.

Author: Done. There was another missing at the bottom too.

> - At times there are links to other docs: `[translate this](don't translate this)`, or specific paragraphs in other parts of the website: `[translate this](don't translate this#translate-this-but-don't-use-capitals)`. Make sure the last part after the hashtag uses the exact same words as the relevant paragraph title you use in your translation.

Reviewer: It's probably worth _real_ examples here, at least explaining that `translate-this-but-don't-use-capitals` is the `<h3>Translate This but DON'T Use Capitals</h3>` heading on `https://jamulus.io/wiki/don%27t%20translate%20this` page titled `translate this`.

Author: Hmm, I'm not particularly smart but I had to read your example several times before I could understand it. My reasoning is that the examples look exactly like they will when translators come across them. These links have caused no end of confusion in the past and I think that after doing a few you become familiar with how they're done - and what each part is.

BTW, I added the fact that apostrophes shouldn't be used either.

Reviewer: Reading it a second time, it seems fine now.

- Some documents contain links to screenshots. Make sure you change the language code in them to your own. You can create screenshots for your language in `/assets/img/[LANG]-screenshots` - note that we do not store these image locally but instead use include files calling an external URL to them. Please see README.md on the website repository for further information. Also note that screenshots that do not need to be localised are symlinked to the English version from each `[LANG]-screenshots` directory.
- There are documents containing portions of text that do not require translation, e.g. scripts, links, etc. In these cases, please do not leave the translation field empty, but copy the source text into the translation field.

> - Pull Requests for translations should have a title which looks like this: `Update [lang] web translation for [release number]`

Reviewer: ```suggestion
- If editing the po file directly, rather than using Weblate, please make sure your Pull Requests for translations have a title which looks like this: `Update [lang] web translation for [release number]`
```

Author: Done.

Reviewer 2: - If editing the .po file directly, don't use Weblate, and give your pull requests for translations a title like: `Update [lang] web translation for [release number]`

Reviewer: That's not the sense here.

> - If editing the po file directly, rather than using Weblate, please make sure your pull requests for translations should have a title which looks like this: `Update [lang] web translation for [release number]`.

Reviewer 2: ```suggestion
- If editing the .po file directly, rather than using Weblate, give your pull requests for translations a title that looks like this: `Update [lang] web translation for [release number]`.
```

Author: Done
